pycqed/tests/instrument_drivers/physical_instruments/test_QuTechCC.py

```diff
@@ -0,0 +1,65 @@
+import unittest
```

Comment:
This belongs in the test folder and not in the physical instruments folder. Is this test hardware dependent? Cause then it should contain some other wrapping as well. We can discuss this.

Reply:
it *is* in the tests folder, see previous comment